hnn_core/extracellular.py

```diff
@@ -516,14 +517,39 @@ def _build(self, cvode=None, include_celltypes='all'):
 
         # contributions of all segments on this rank to total calculated
         # potential at electrode (_PC.allreduce called in _simulate_dipole)
-        self._nrn_voltages = h.Vector()
+        # NB voltages of all contacts are initialised to 0 mV, i.e., the
+        # potential at time 0.0 ms is defined to be zero.
+        self._nrn_voltages = h.Vector(self.n_contacts, 0.)
```

Comment:
Ah, I now get why the second argument (i.e., `0.`) is important!

Reply:
```suggestion
        self._nrn_voltages = h.Vector(self.n_contacts, init=0.)
```

to be clear from the code :)

Reply:
Keyword-arguments in Neuron? Let me check... Nope, not a thing ;)